docs/source/changelog/0.6.0-changelog.rst

```diff
@@ -0,0 +1,439 @@
+   - Removed the ``--sound`` option. It lost support long ago with the removal of SoX.
+
+Deprecated classes and functions
+--------------------------------
+
+* `#1354 <https://github.com/ManimCommunity/manim/pull/1354>`__: Refactored and deprecated a few functions in space_ops.py
+
+
+* `#1370 <https://github.com/ManimCommunity/manim/pull/1370>`__: Remove TexMobject and TextMobject
+   Deleted TexMobject and TextMobject
+
+* `#1349 <https://github.com/ManimCommunity/manim/pull/1349>`__: Removed the deprecated Small Dot
+
+
+New features
+------------
+
+* `#1386 <https://github.com/ManimCommunity/manim/pull/1386>`__: Implement utility methods for adding/removing vertices and edges of graphs; allow custom mobjects as vertices
+
+
+* `#1385 <https://github.com/ManimCommunity/manim/pull/1385>`__: Added :meth:`~.CoordinateSystem.get_line_graph` for plotting a line graph
+   Added :meth:`~.CoordinateSystem.get_line_graph` that returns a line graph from lists of points along x, y and z (optional) axes.
+
+* `#1381 <https://github.com/ManimCommunity/manim/pull/1381>`__: Hot reloading for the OpenGL renderer
+   Rerun scene when the input file is modified
+
+* `#1383 <https://github.com/ManimCommunity/manim/pull/1383>`__: Overhaul of the `animation.indication` module interfaces
+
+
+* `#1348 <https://github.com/ManimCommunity/manim/pull/1348>`__: Added :class:`Polyhedron`, and platonic solids :class:`Tetrahedron`, :class:`Octahedron`, :class:`Icosahedron` and :class:`Dodecahedron`
+
+
+* `#1285 <https://github.com/ManimCommunity/manim/pull/1285>`__: Add :meth:`~.Scene.interactive_embed` for OpenGL rendering
+   :meth:`~.Scene.interactive_embed` allows interaction with Scene via mouse and keyboard as well as dynamic commands via an iPython terminal.
+
+* `#1261 <https://github.com/ManimCommunity/manim/pull/1261>`__: Render image automatically if no animation is played in a scene
+
+
+Enhancements
+------------
+
+* `#1398 <https://github.com/ManimCommunity/manim/pull/1398>`__: Fix and enhance `Mobject.arrange_in_grid`
+   `arrange_in_grid` now actually arranges submobjects in a grid. Added new parameters `buff`, `cell_alignment`, `row_alignments`, `col_alignments`, `row_heights`, `col_widths`, `flow_order`.
+
+* `#1407 <https://github.com/ManimCommunity/manim/pull/1407>`__: Fix bug in :meth:`vector_coordinate_label` and move it to :class:`~.geometry.py`
+
+
+* `#1380 <https://github.com/ManimCommunity/manim/pull/1380>`__: Allow image objects as background images
+
+
+* `#1391 <https://github.com/ManimCommunity/manim/pull/1391>`__: Add `path_arc` support to `.animate` syntax
+   The parameter `path_arc` of :class:`~.Transform` now works with the `.animate` syntax
+
+* `#1364 <https://github.com/ManimCommunity/manim/pull/1364>`__: Added :meth:`~.Mobject.match_points`
+   - Added :func:`~.Mobject.become_points`, which transforms the points, positions and submobjects of a Mobject to match that of the other while keeping style unchanged.
```

Comment:
CC @Darylgolden 
```suggestion
   - Added :meth:`~.Mobject.match_points`, which transforms the points, positions and submobjects of a Mobject to match that of the other while keeping style unchanged.
```

Reply:
LGTM

Reply:
can you change it in the PR title, then behackl would render this again?

Reply:
I've already taken care of it. 👍